### ISSUE TITLE
Interface Clarity

### ISSUE THREAD

**Issue by SilentFang** [smspillane/QuizMaster#5]
It can be sometimes tricky to work out was was new and old content in the terminal window. Possible using something like "curses" would help with building a UI.

**Comment by lberezy** [comment]
I'm fairly sure he's just learning formatted strings. Curses...yeah, that might be a bit much.

That said, apparently if you do `print(chr(27) + "[2J")` that will [send the correct ANSI escape code to the terminal to clear the screen](http://stackoverflow.com/questions/2084508/clear-terminal-in-python).

**Comment by SilentFang** [comment]
Yeah, on second though curses is without a doubt overkill, it's just the first thing that came to mind when thinking about making interactive CLI programs like this.
